base/main.tf

```diff
@@ -81,9 +81,13 @@ data "template_file" "helm_values" {
 }
 
 resource "null_resource" "helm_values_file" {
+  triggers {
+    content = "${data.template_file.helm_values.rendered}"
+  }
+
   provisioner "local-exec" {
     command = <<-EOC
-      tee helm_values.yaml <<EOF
+      tee ${path.cwd}/helm_values.yaml <<EOF
```

Comment:
Will this not give problems if somebody else runs this? e.g. my path can be different from yours.